package-lock.json

```diff
@@ -18600,8 +18600,8 @@
           "integrity": "sha1-OpoguEYlI+RHz7x+i7gO1me/xVI=",
           "dev": true,
           "requires": {
-            "errno": "^0.1.3",
-            "readable-stream": "^2.0.1"
+            "errno": "0.1.7",
+            "readable-stream": "2.3.6"
```

Comment:
Probably the packages-lock.json must be regenerated after rebase it with advanced-permissions-phase-2 and upgrade to npm 6

Reply:
@jespino Yes I'll regen. in long-lived feature branch.